src/javascript/lib/landing_pages/ico.js

```diff
@@ -103,6 +104,71 @@ function clearHash() {
     }
 }
 
+function signUpInit() {
+    var el_email   = document.getElementById('email');
+    var el_signup  = document.getElementById('signup');
+    var el_success = document.getElementById('success');
+
+    var ws = wsConnect();
+
+    function sendVerifyEmail() {
+        wsSend(ws, {
+            verify_email: el_email.value,
+            type        : 'account_opening'
+        });
+    }
+
+    var validation_set = false; // To prevent validating before submit
+
+    document.getElementById('frm_verify_email').addEventListener('submit', function (evt) {
+        evt.preventDefault();
+
+        if (!validateEmail(el_email.value)) {
+            if (!validation_set) {
+                ['input', 'change'].forEach(function (evt) {
+                    el_email.addEventListener(evt, function () {
+                        setValidationStyle(el_email, !validateEmail(el_email.value));
+                    });
+                });
+                setValidationStyle(el_email, !validateEmail(el_email.value));
+                validation_set = true;
+            }
+            return false;
+        }
+
+        if (ws.readyState === 1) {
+            sendVerifyEmail();
+        } else {
+            ws.onopen = sendVerifyEmail;
+        }
+    });
+
+    ws.onmessage = function(msg) {
+        var response = JSON.parse(msg.data);
+        setValidationStyle(el_email, response.error);
+        console.log(response);
```

Comment:
@zakibinary  LGTM :+1:

An is just one `console.log` statement left here.

Reply:
whoops